src/main.c

```diff
@@ -17,6 +17,14 @@ extern void vt100_screen_rows;
 #pragma zpsym ("vt100_screen_cols");
 #pragma zpsym ("vt100_screen_rows");
 
+#if defined(__ATARI__)
+bool trip=false;       /* Used for PROCEED interrupt. */
```

Comment:
this is only defined for ATARI, but you later use it in a block for ATARI or C64.
I don't think this compiles.